next.config.js

```diff
@@ -1,4 +1,6 @@
-module.exports = {
+const { withPlausibleProxy } = require('next-plausible');
+
+module.exports = withPlausibleProxy({
```

Comment:
hmm shouldn't we call it like `module.exports = withPlausibleProxy(){...`?
https://plausible.io/docs/proxy/guides/nextjs#using-next-plausible